modules/lndr_debug/lndr.debug.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @file
+ * Code for the Lndr debug functions.
+ */
+
+/**
+ * Record and debug the incoming request.
+ */
+function lndr_example_service_debug() {
+drush_print_r(__FUNCTION__);
+
+
+  $debug = array(
+    'date time' => date(('Y-m-d H:i:s')),
+    'request_headers' => getallheaders(),
```

Comment:
See the lndr.module, if we are using this here we might also need to re-define the module for nginx servers if(!function_exists('getallheaders')) {
...
}

Reply:
lndr_debug module has a dependency to lndr module, so lndr.module file will be included before lndr.debug.inc